zwave_js_server/const/__init__.py

```diff
@@ -3,14 +3,15 @@
 from enum import Enum, IntEnum
 
 # minimal server schema version we can handle
-MIN_SERVER_SCHEMA_VERSION = 17
+MIN_SERVER_SCHEMA_VERSION = 18
 # max server schema version we can handle (and our code is compatible with)
-MAX_SERVER_SCHEMA_VERSION = 17
+MAX_SERVER_SCHEMA_VERSION = 18
 
 TYPING_EXTENSION_FOR_TYPEDDICT_REQUIRED = sys.version_info < (3, 9, 2)
 
 VALUE_UNKNOWN = "unknown"
 
+NOT_INTERVIEWED = "None"
```

Comment:
Should we create an enum with the possible interview stages?

https://zwave-js.github.io/node-zwave-js/#/api/node?id=interviewstage

Reply:
When I last spoke to Al about this, he suggested not to because the interview stages were subject to change at any time (they are not part of the standard I don't think). I think `None` and `Failed` are safe to remain which is why I was comfortable creating constants for them

Reply:
Ok.